src/stringIndices.js

```diff
@@ -0,0 +1,11 @@
+export default function stringIndices(string, index) {
+
+  var removeSpecialChar = string.replace(/\W+/g, ' ')
+  var strToArr = removeSpecialChar.split(' ')
+
+  if(index > strToArr.length || index <= 0) {
+    return ''
+  } else {
+    return strToArr[index - 1]
```

Comment:
to make this function clearer, consider changing the name of the parameter index.  Maybe something like wordNumber, that way strToArr[wordNumber -1] clearly refers to the desired index of strToArr.